src/Cake.Common/Xml/XmlPokeAliases.cs

```diff
@@ -256,6 +256,44 @@ public static void XmlPoke(this ICakeContext context, FilePath filePath, string
         /// <param name="xpath">The xpath of the nodes to set.</param>
         /// <param name="value">The value to set too. Leave blank to remove the selected nodes.</param>
         /// <returns>Resulting XML.</returns>
+        /// <remarks>
+        /// The method is now obsolete replaced by <see cref="XmlPokeString(Cake.Core.ICakeContext,string,string,string)"/>
+        /// </remarks>
+        [CakeMethodAlias]
+        [Obsolete("Use XmlPokeString alias instead", false)]
```

Comment:
Can we change this message to be more in line with the other obsolete messages?

`Please use XmlPokeString(string, string, string) instead.`